decidim-elections/lib/decidim/votings/admin_engine.rb

```diff
@@ -33,6 +33,7 @@ class AdminEngine < ::Rails::Engine
               get :status, action: :status
               post :create, action: :create
               put :update, action: :update
+              get :generate_access_codes, action: :generate_access_codes
```

Comment:
💬 why do we need to specify the action on every route here? 🤔

Reply:
'cause I'm a copy-paste master hehe
They're not really necessary, will remove them in https://github.com/decidim/decidim/pull/7756